core-addon/telemetry/LegacyTelemetryApi.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
We *could* alter the `browser.telemetry.submitEncryptedPing` API to accommodate this, but since we're moving to glean eventually I think this is a fine approach.

Reply:
Yes, I was thinking the same: we will eventually get rid of that API or change it in Firefox next year (regardless of Ion). Moreover, changing the API this way has the nice advantage of not having to ride the trains :)